api/graphite.go

```diff
@@ -847,6 +850,11 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan *expr.Plan)
 		resolveSeriesRequests[strippedreq] = append(resolveSeriesRequests[strippedreq], plan.Reqs[i])
 	}
 
+	// Output to see how useful this optimization might be
+	if len(resolveSeriesRequests) != len(plan.Reqs) {
+		log.Infof("executePlan: traceID=%s Making %d roundtrips to resolve %d reqs", traceID, len(resolveSeriesRequests), len(plan.Reqs))
```

Comment:
not sure if this deserves its own log line.
could this be a metric instead? or just a stat logged at the end?

Reply:
I think this line can be removed entirely. It's not all that useful and it's mostly suppressed by the similar optimization [here](https://github.com/grafana/metrictank/blob/9350203dd915facc5981f90701f39ef7228f74b3/expr/plan.go#L182)

Reply:
ok let's remove